drivers/cmd/gcloud-storage/main_test.go

```diff
@@ -0,0 +1,91 @@
+package main
```

Comment:
Please add the comment like below at the top so it only gets executed once in CI

https://github.com/honeydipper/honeydipper/blob/4845291926c9c3c76a3be115860ce56383afffd8/internal/workflow/session_parallel_test.go#L7